mne/io/kit/kit.py

```diff
@@ -504,7 +504,7 @@ def _set_dig_kit(mrk, elp, hsp):
         hsp = _read_dig_points(hsp)
     n_pts = len(hsp)
     if n_pts > KIT.DIG_POINTS:
-        hsp = _decimate_points(hsp, res=5)
+        hsp = _decimate_points(hsp, res=0.005)
```

Comment:
I think it makes sense for it to be unit-agnostic as it currently is, since you enter both the values and the grid spacing.